model/qti/Element.php

```diff
@@ -126,10 +161,10 @@ public function removeAttributeValue($name)
      * Set the attributes for the the Qti Element
      * Argument format: array(attributeName => value)
      *
-     * @param array $values
-     * @throws InvalidArgumentException
+     * @throws AttributeException
+     * @throws QtiModelException
      */
-    public function setAttributes($values)
+    public function setAttributes(array $values): void
```

Comment:
This might break, unfortunately :/ 

Cause internally this method checks is parameter is an array, so typehint is dangerous. Can you please check the risk to add this?

Reply:
I was thinking that I should even remove `else` from this method and keep parameter typed.